spec/components/contents_list_with_body_spec.rb

```diff
@@ -0,0 +1,56 @@
+RSpec.describe "ContentsListWithBodyComponent", type: :view do
+  def component_name
+    "contents_list_with_body"
+  end
+
+  def contents_list
+    [
+      { href: "/one", text: "1. One" },
+      { href: "/two", text: "2. Two" },
+    ]
+  end
+
+  def block
+    "<p>Foo</p>".html_safe
+  end
+
+  it "renders nothing without a block" do
+    render_component(contents: contents_list)
+    expect(rendered).to be_empty
```

Comment:
Could you add a space before the first expectation here? (a sort of non-lintable convention we have)